quizsol7.ipynb

```diff
@@ -0,0 +1,78 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "852ba695-91a5-40cc-aa99-a34bcdb0e201",
+   "metadata": {},
+   "source": [
+    "7.Write a program to create \"intro.txt\" file.  Print only even number of row."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "cbbe124c-442e-420c-bcc8-d375c2db7e9f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import csv "
```

Comment:
write a other way to import a file ?

Reply:
we can write as "from csv import reader" ....!

Reply:
yes